js/classes/inventory.js

```diff
@@ -1,44 +1,95 @@
 /**
  * The Inventory class. Will be used to keep track of the player's inventory.
+ * Holds ammo, food, and runes.
+ * 
+ * If we deem the duplicate methods for each item type to be unnecessary, we can combine them. 
+ * However, for scalability I decided to split them up.
```

Comment:
You're right that some of them (e.g. the adjust methods) could be generalized. It doesn't make sense to me to try and combine all of them though.